packages/web-wallet/src/hooks/use-toast.ts

```diff
@@ -0,0 +1,193 @@
+"use client"
```

Comment:
Why are we using this directive, isn't this a static SPA, meaning it's all client side?